awx/main/tests/functional/api/test_job_template.py

```diff
@@ -217,7 +225,7 @@ def test_jt_admin_copy_edit_functional(jt_copy_edit, rando, get, post):
 
     post_data = get_response.data
     post_data['name'] = '%s @ 12:19:47 pm' % post_data['name']
-    post_response = post(reverse('api:job_template_list'), user=rando, data=post_data)
+    post_response = post(reverse('api:job_template_list'), user=rando, data=dict(post_data))
```

Comment:
`post_data` appears to already be a dict, why do you have to cast it again? Or is it not really a dict?

Reply:
from memory, it was a Django `ResponseDict` or something like that. This does bring up pained philosophical questions about type hints, like, does it allow subclasses of the expected type? But for the purposes of this particular fixture, I think I'm fine forcing all callers to cast to a plain dict.